debezium-connector-mariadb/src/main/java/io/debezium/connector/mariadb/MariaDbConnectorConfig.java

```diff
@@ -72,6 +72,12 @@ public enum SnapshotLockingMode implements EnumeratedValue {
          * set to schema_only or schema_only_recovery.
          */
         NONE("none"),
+
+        /**
+         * the connector holds the global read lock but does not set a transaction `START TRANSACTION WITH CONSISTENT SNAPSHOT` to ensure
+         * that the snapshot process will have a view from the mvcc. This mode is valid for myRocks engine that doesn't provide mvcc support
+         */
+        AT_LEAST_ONCE("at_least_once"),
```

Comment:
Do we need new snapshot mode? Should not this be done via `snapshot.locking.mode`? I can envision name like `minimal_at_least_once`.

Reply:
don't worry! let me know if it's reasonable now moved in `binlog` module :rocket: